movie/src/components/Navbar/Navbar.css

```diff
@@ -1,24 +1,29 @@
 .navbar {
-  width: 70%;
+  max-width: 1400px;
+  min-width: 420px;
+  width: 80%;
   margin: auto;
   display: flex;
   align-items: center;
   justify-content: space-between;
-  box-sizing: border-box;
 }
 
 .leftNav {
+  background-color: rgb(3, 25, 96);
+  top: 0;
+  right: 0;
```

Comment:
why do we need `top` and `right` for this?

Reply:
Sometimes, I copy from another menu style on internet. I forgot to change in order to match with my menu. I will consider on it later.